### PR TITLE
Return promise function to org queue loading component

### PR DESCRIPTION
**Pull request by lowellrex**
Completes the work started in #10912 in preparation for #10049 by moving the loading promise back to the `OrganizationQueueLoadingScreen` component.

### CODE REVIEW

File: client/app/queue/OrganizationQueueLoadingScreen.jsx [hunk 1]
```diff
@@ -17,14 +18,34 @@ import {
 class OrganizationQueueLoadingScreen extends React.PureComponent {
   reload = () => window.location.reload();
 
+  createOrgQueueLoadPromise = () => ApiUtil.get(this.props.urlToLoad, { timeout: { response: 5 * 60 * 1000 } }).
```
Comment (kevmo) [inline client/app/queue/OrganizationQueueLoadingScreen.jsx]: Why not just write `300000`?

Comment (lowellrex) [reply client/app/queue/OrganizationQueueLoadingScreen.jsx]: To me `5 * 60 * 1000` reads more clearly as 5 minutes than `300000` (milliseconds) does.

Comment (kevmo) [reply client/app/queue/OrganizationQueueLoadingScreen.jsx]: That's fair.

File: spec/models/task_spec.rb [hunk 2]
```diff
@@ -695,7 +695,7 @@ def next_assignee(_options = {})
 
     it "returns and destroys related timers" do
       expect(TaskTimer.where(task_id: task_id).count).to eq(task_timer_count)
-      expect(task.task_timers).to eq(task_timers)
+      expect(task.task_timers.to_a).to match_array(task_timers)
```
Comment (kevmo) [inline spec/models/task_spec.rb]: :+1:

**Pull request merged**